src/options_parser.cc

```diff
@@ -42,14 +42,13 @@ OptionsResult* protagonist::ParseOptionsObject(Handle<Object> optionsObject) {
         }
         else if (TypeOptionKey == *String::Utf8Value(key)) {
             // TypeOption
-            const char *strValue = *String::Utf8Value(value);
```

Comment:
It is actually important to understand why this was the issue. Basically you were relying on a compiler's implementation. Assigning a temporary value to `const char *strValue` which, in some implementation didn't have to be there when you were dereferencing the `strValue` .